Automattic/pocket-casts-ios
# Icon: set the red icon as default

Set the red icon as the default one.

## To test

I recommend having two Xcodes opened, one with `release/7.41` and another one with this branch.

1. Run the 7.41 version
2. Check that Pride 2023 is the default icon
3. Run this branch
4. ✅ The red default icon should be the default icon
5. Go to Settings > Appearance
6. ✅ Default should be the selected icon
7. Remove the app
8. Run 7.41 again
9. Change the default icon
10. Run this branch
11. ✅ Your previously selected icon should be the home screen icon
5. Go to Settings > Appearance
6. ✅ The previously selected icon should be the selected one

## Checklist
